SQL/New_patches/2018-01-17_normalisation_visit.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE `visit` (
+  `VisitID` int(10) unsigned NOT NULL auto_increment,
+  `VisitName` varchar(100) NOT NULL,
+  CONSTRAINT `visit_PK` PRIMARY KEY (`VisitID`),
+  CONSTRAINT `visit_name_UK` UNIQUE KEY (`VisitName`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+CREATE TABLE `visit_project_subproject_rel` (
+  `VisitID` int(10) unsigned NOT NULL,
+  `ProjectID` int(2) NOT NULL,
+  `SubprojectID` int(10) unsigned NOT NULL,
+  CONSTRAINT visit_project_subprojet_rel_PK PRIMARY KEY (`VisitID`, `ProjectID`, `SubprojectID`),
```

Comment:
Same here

Reply:
corrected, keep using the french spelling.